etc/trunion-dev.ini

```diff
@@ -80,3 +80,23 @@ certfile = trunion/tests/test_crt.jwk
 chainfile = trunion/tests/test_x509_chain.pem
 permitted_issuers = https://marketplace-dev.allizom.org
 we_are_signing = receipts
+
+[addons]
+ca_cert_file = /etc/trunion/addons_root_ca_cert.pem
+ephemeral_key_size = 2048
```

Comment:
In bits.  I don't see a need to document it since key sizes are commonly known to ops folks to be in bits.